### PR TITLE
[AIRFLOW-1543] Improve error message for incorrect fernet_key

### PR DESCRIPTION
**Pull request by gtt116**
Dear Airflow maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. 
    - https://issues.apache.org/jira/browse/AIRFLOW-1543

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:
No UI changes, only some improvement for logging.

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
Just some trivial modification, no tests add, if necessary it could be done.

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

File: airflow/models.py [hunk 1]
```diff
@@ -3824,7 +3824,7 @@ def __repr__(self):
         return self.label
 
 
-class Variable(Base):
+class Variable(Base, LoggingMixin):
```
Comment (ashb) [inline airflow/models.py:3827]: This seems like an un-related change....? Is it needed?

**Comment by tkaymak** [comment]
any update on this?

**Comment by ron819** [comment]
@gtt116 can you rebase?

**Comment by ashb** [comment]
Fixed elsewhere

**Pull request closed without merging**